src/core/dbus-mount.c

```diff
@@ -57,6 +85,7 @@ const sd_bus_vtable bus_mount_vtable[] = {
         BUS_EXEC_COMMAND_VTABLE("ExecMount", offsetof(Mount, exec_command[MOUNT_EXEC_MOUNT]), SD_BUS_VTABLE_PROPERTY_EMITS_INVALIDATION),
         BUS_EXEC_COMMAND_VTABLE("ExecUnmount", offsetof(Mount, exec_command[MOUNT_EXEC_UNMOUNT]), SD_BUS_VTABLE_PROPERTY_EMITS_INVALIDATION),
         BUS_EXEC_COMMAND_VTABLE("ExecRemount", offsetof(Mount, exec_command[MOUNT_EXEC_REMOUNT]), SD_BUS_VTABLE_PROPERTY_EMITS_INVALIDATION),
+        SD_BUS_METHOD("Remount", "s", NULL, bus_mount_method_remount, SD_BUS_VTABLE_UNPRIVILEGED),
```

Comment:
Not sure, but I think it is better to take `a(sv)`?

Reply:
To allow more than one option string? How is that handled with mount units?

Reply:
I thought that it may be better to take other properties (not only defined in dbus-mount.c but also defined in dbus-execute.c and dbus-unit.c). But I guess it can be done by `SetUnitProperties` + `Remount`. So, it may not be necessary.